### PR TITLE
DM page empty state

### PR DESCRIPTION
**Pull request by Will-Howard**
See [designs](https://www.figma.com/file/d09iVzNSpZp9GyWGcfkWmX/Q2-2023?node-id=2582%3A1802&mode=dev) here

The design given was for the state when there are no messages at all, but I realised this essentially can't happen because we send a welcome message as soon as someone signs up. I thought this version looked a lot better even in the case where you have messages but haven't selected one, so I have made it for that case instead (which just required changing "No messages yet" to "No conversation selected")

Screenshot 2023-12-05 at 15 40 45
Screenshot 2023-12-05 at 15 40 51

cc @agnesstenlund

### CODE REVIEW

File: packages/lesswrong/components/messaging/FriendlyInbox.tsx [hunk 1]
```diff
@@ -51,14 +51,22 @@ const styles = (theme: ThemeType): JssStyles => ({
     minHeight: 0,
     display: "flex",
     flexDirection: "row",
-    flex: 1
+    flex: 1,
+    overflow: "hidden", // to simplify border radius
+    border: theme.palette.border.grey200,
+    borderRadius: `${theme.borderRadius.default}px ${theme.borderRadius.default}px 0px 0px`,
+    backgroundColor: theme.palette.background.pageActiveAreaBackground,
```
Comment (Will-Howard) [inline packages/lesswrong/components/messaging/FriendlyInbox.tsx:58]: The way I had done the borders before was less than ideal, I've simplified it here (mainly moved the outer border to the `table` class)

File: packages/lesswrong/components/messaging/FriendlyInbox.tsx [hunk 2]
```diff
@@ -136,7 +124,47 @@ const styles = (theme: ThemeType): JssStyles => ({
...
+    margin: "auto auto",
+    alignItems: "center",
+    width: "fit-content",
+    // The below two lines make it look more visually centered
+    position: "relative",
+    bottom: "10%",
+  },
+  emptyStateIcon: {
+    width: 100,
+    height: 100,
```
Comment (Will-Howard) [inline packages/lesswrong/components/messaging/FriendlyInbox.tsx:142]: I made this slightly bigger than the design

File: packages/lesswrong/components/messaging/FriendlyInbox.tsx [hunk 3]
```diff
@@ -241,27 +270,43 @@ const FriendlyInbox = ({
...
-          </div>
-          <div className={classes.conversation} ref={selectedConversationRef}>
-            {selectedConversation ? (
-              <>
-                <Link to="/inbox" className={classes.backButton}> Go back to Inbox </Link>
+          {!!selectedConversation && (
+            <>
+              <div className={classes.columnHeader}>
+                <div className={classes.headerText}>{title}</div>
+                {conversationId ? (
```
Comment (jpaddison3) [inline packages/lesswrong/components/messaging/FriendlyInbox.tsx:277]: How do you get to have a selected conversation without a conversation id?

**Review by jpaddison3 — APPROVED** [review]

**Comment by jpaddison3** [comment]
This looks really sweet, nice work @Will-Howard and @agnesstenlund

**Comment by agnesstenlund** [comment]
sorry late reply, I love it ✨

**Comment by agnesstenlund** [comment]
oh tiny thin, `forum` should have a big F, we could add that to another follow-up PR?

**Pull request merged**